### PR TITLE
Change token name for approval and disable approval when the balance is too low

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- Change token approval to specify the correct token
- Disable approve button when balance is too low

### CODE REVIEW

File: pages/[id]/bridge.tsx [hunk 1]
```diff
@@ -204,7 +211,7 @@ const Bridge = () => {
   // Define new variables that help control UI states and display pretty error strings
   const { errorType, errorReason } = formatError(error);
   // Helpers for different parts of UI state
-  const isAmountError = errorType === ErrorType.ERC20AmountError;
+  const isAmountError = errorType === ErrorType.ERC20AmountError || !isSufficientBalance;
```
Comment (wildmolasses) [inline pages/[id]/bridge.tsx:214]: in the case of `!isSufficientBalance`, the `errorType` is already `ErrorType.ERC20AmountError` so we can make this shorter

```suggestion
  const isAmountError = errorType === ErrorType.ERC20AmountError;
```

File: pages/[id]/bridge.tsx [hunk 2]
```diff
@@ -68,6 +68,7 @@ const Bridge = () => {
   });
 
   const needsAllowanceL1 = (allowanceL1 || 0) < rawAmount;
+  const isSufficientBalance = (source.token?.value || 0) >= rawAmount;
```
Comment (wildmolasses) [inline pages/[id]/bridge.tsx:71]: [arbitrary place to leave comment]

functionality currently not working when i have 0 balance. Asks me to set approval, no error displayed (although, red error icon displays in input box)

**Review by wildmolasses — CHANGES_REQUESTED** [review]

File: pages/[id]/bridge.tsx (hunk 1 above)
Comment (alexkeating) [reply pages/[id]/bridge.tsx]: https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/129/commits/1f438166e417b47324f54d81c9704c19638407b4

File: pages/[id]/bridge.tsx (hunk 2 above)
Comment (alexkeating) [reply pages/[id]/bridge.tsx:71]: https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/129/commits/1f438166e417b47324f54d81c9704c19638407b4

**Review by wildmolasses — APPROVED** [review]
lgtm

**Pull request merged**